### PR TITLE
Unit Test -  Test Home View Model

### PR DESCRIPTION
**Pull request by at-toannguyen1**
#### Related issues (Required)
- Task #191
- Bug #

#### Description (Required)
- [x] Init Test Home View Model

### CODE REVIEW

File: app/src/test/java/vn/asiantech/way/HomeViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Test
+    fun `Given tracking history - When get tracking history - Then return right list tracking history`() {
+        val trackingHistory = mutableListOf<TrackingInformation>()
+        val test = TestObserver<MutableList<TrackingInformation>>()
+        `when`(assetDataRepository.getTrackingHistory()).thenReturn(trackingHistory)
+        viewModel.getTrackingHistory().subscribe(test)
+        test.assertValue { it == trackingHistory }
+    }
+}
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/HomeViewModelTest.kt]: add break line

File: app/src/test/java/vn/asiantech/way/HomeViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    private lateinit var viewModel: HomeViewModel
+
+    @Before
+    fun initTest() {
+        MockitoAnnotations.initMocks(this)
+        viewModel = HomeViewModel(assetDataRepository)
+    }
+
+    @Test
+    fun `Given tracking history - When get tracking history - Then return right list tracking history`() {
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/HomeViewModelTest.kt]: change method name to `Given a list tracking history - When get tracking history - Then return right list tracking history`

File: app/src/test/java/vn/asiantech/way/HomeViewModelTest.kt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * HomeViewModelTest.
+ *
+ * @author at-ToanNguyen
+ */
+@Suppress("IllegalIdentifier")
+class HomeViewModelTest {
+    @Mock
+    private lateinit var assetDataRepository: LocalRepository
+
```
Comment (at-vinhhuynh) [inline app/src/test/java/vn/asiantech/way/HomeViewModelTest.kt]: remove this line

**Comment by at-vinhhuynh** [comment]
@at-tienhoang 
I finish review this PR, please check it again.
thank you!

**Comment by at-dev-support-01** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please rebase to get rid of the merge commits in this PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">All test passed. (9/9)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**